### ISSUE TITLE
NVDA mouse tracking fails in Canary builds of Microsoft Edge

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#10459]
### Steps to reproduce:
1. From NVDA's mouse settings, enable mouse tracking.
2. Install latest build of Chromium-based Microsoft Edge Canary.
3. Launch Microsoft Edge Canary and load a page in it (it does not matter which - nvaccess.org will do).
4. Move the mouse cursor over some objects (e.g. text blocks, links, etc.).

### Actual behavior:
NVDA stays silent.

### Expected behavior:
NVDA should recognize and read the text in the objects under the mouse cursor.

### System configuration
- Operating system: Windows 10 Pro version 1903 (build 18362.449), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-19065,1b7cddfd, in bulgarian.
- Microsoft Edge version: Microsoft Edge Canary 80.0.320.0 (official build), 64-bit, in bulgarian.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

#### Does the issue still occur after restarting your PC?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - NVDA 2019.2.1. The behavior is the same.

**Comment by leonardder** [comment]
Can you verify the following things?


* Does this also apply to Chrome Canary 80?

* When getting developer info for the object, are the objects in Edge 
Canary IAccessible or UIA objects?

* How is this in Edge Dev/Beta?

**Comment by k-kolev1985** [comment]
@leonardder Sorry, I cannot verify any of those, due to the following reasons:
- I couldn't find a portable copy of Chrome Canary 80 and I prefer not to install Chrome on my machine if possible. If needs be however, I could install it on a virtual machine.
- I don't have the knowledge to determine if the objects are iAccessible or UIA ones.
- I have only Microsoft Edge Canary  installed on this machine. I don't know if I can install dev or beta on it as well without removing the Canary one. I could install dev or beta on a virtual machine however, if necessary.

I think however, that all started with v80 of Microsoft Edge Canary.

**Comment by leonardder** [comment]
I will investigate with Edge Canary later today.

**Comment by dnz3d4c** [comment]
I remember Chrome's mouse tracking on NVDA wasn't smooth.

1. Use the object navigation feature to navigate to the desired object, then press NVDA + keyPadSlash. Mouse object and navigation object are not the same.
2. I don't remember exactly which version of Chrome, but I experienced NVDA hangs when browsing web pages displayed in Chrome with the mouse.

**Comment by gdata1** [comment]
I can confirm that, too. but when you turn on the Native UI Automation flag, mouse navigation gets better.

**Comment by leonardder** [comment]
@k-kolev1985 how is this now?

**Comment by k-kolev1985** [comment]
@leonardder You mean after enabling the Native UI Automation flag,? I haven't tried this. I may try it these days and report back.

**Comment by gdata1** [comment]
@leonardder I have improvements after turning on this flag, but I turned it off because I can't use it with the keyboard. Do not work cosing keyboards.

**Comment by leonardder** [comment]
> @leonardder You mean after enabling the Native UI Automation flag,?

Nope, just according to your steps to reproduce in the initial comment of this issue.
Also, are there differences between how Edge and Chrome behave with this regard?

**Comment by k-kolev1985** [comment]
The behavior is the same with the latest alpha build of NVDA and latest Microsoft Edge Canary. I can get exact versions of both if necessary.

I don't have Google Chrome installed on my machines - only a portable copy for test purposes. But even it is a stable 78 release. With it NVDA's mouse tracking behaves normally.

**Comment by k-kolev1985** [comment]
I don't know if the bug is the same or just a similar one, but mouse tracking is not working again in the Chromium-based Microsoft Edge. When I point the mouse cursor over the page content, NVDA says only "pane" and nothing else. I even tried with Native UI Automation disabled, but the result is the same. Also tried with add-ons disabled - same result.

Test environment:
- Operating system: Windows 10 Pro version 1909 (build 18363.752), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-19852,c6b6904b, in bulgarian.
- Chromium-based Microsoft Edge version: 83.0.470.0, in bulgarian.
- Processor: Intel Core i5-9400F at 2.90GHz.
- RAM Memory: 16.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

**Comment by k-kolev1985** [comment]
Never mind my last comment - it is fixed now. I see that there was an update to Microsoft Edge Canary today - that one probably contained the fix. Well, that is the risk of running test versions of programs I guess... 😀

**Comment by gdata1** [comment]
on some sites, I have a serious problem navigating with a mouse in the new edge of all versions. even Canary. For example. the site www.vk.com very poorly accessible with the mouse.

**Comment by feerrenrut** [comment]
@k-kolev1985 Are you saying that this issue is resolved and we can close it?

**Comment by k-kolev1985** [comment]
@feerrenrut Yes, the problem is fixed (at least for now). Let's hope it stays that way. 😀 You can close the bug.
